docs/site/content/docs/latest/tips.md

```diff
@@ -0,0 +1,20 @@
+# Connect to Cluster Nodes with SSH
+
+You can use SSH to connect to individual nodes of management clusters or workload clusters. The SSH key pair that you created when you deployed the management cluster must be available on the machine on which you run the SSH command. Consequently, you must run `ssh` commands on the machine on which you run `tanzu` commands.
```

Comment:
```suggestion
You can use SSH to connect to nodes in management clusters and workload clusters. The SSH key pair that you created when you deployed the management cluster must be available on the machine on which you run the SSH command.
```

Reply:
some word smithing and recommending the removal of:

>  Consequently, you must run `ssh` commands on the machine on which you run `tanzu` commands.

This is not required, only the ssh key being on the client (host) you're connecting **from**.